docs/topics/device_connection.md

```diff
@@ -0,0 +1,265 @@
+entity itself. As en example, we will provision statically the Reference of the Weather Station and its location.
+
+To provision the device in the platfor we issue the following request to the IoTAgent:
+
+    POST /iot/devices
+    Content-Type: application/json
+    Fiware-service: smartown
+    Fiware-servicepath: /gardens
+
+    {
+      "devices": [
+        {
+          "device_id": "ws1956672",
+          "protocol": "IoTA-UL",
+          "entity_name": "WeatherStation1",
+          "entity_type": "Device",
+          "attributes": [
+            {
+              "object_id": "t",
+              "name": "temperature",
+              "type": "degrees"
+            },
+            {
+              "object_id": "h",
+              "name": "humidity",
+              "type": "percentage"
+            }
+          ],
+          "commands": [
+            {
+              "name": "configuration",
+              "type": "command"
+            }
+          ],
+          "static_attributes": [
+            {
+              "name": "location",
+              "type": "geo:point",
+              "value": "22.1, -17"
+            },
+            {
+              "name": "reference",
+              "type": "string",
+              "value": "917834508965243"
+            }
+          ]
+        }
+      ]
+    }
+
+This will provision our ws1 workstation in the system, making it ready to receive measures.
+
+### Sending the measure
+
+In order to send measures to the platform, the device has to fulfill the following requirements:
+
+* Connectivity to the South Bound of the platform.
+* Either an MQTT or an HTTP client (as those are the transport protocols currently supported).
+
+Being the most common, we will assume that your constraint device have an HTTP client, that can send requests to the
+platform. This interaction with the Platform can be implemented with whichever programming language your device supports.
+No SDKs are provided for south bound interactions.
+
+In this case, a typical measure report from our Weather Station will consist on a request like the following
+one:
+
+    POST /iot/d?k=801230BJKL23Y9090DSFL123HJK09H324HV8732&i=ws1956672
+    Content-Type: text/plain
+
+    t|22|h|78
+
+You should notice several things in this request:
+
+* First of all, the path is '/iot/d'. The notification path will depend on the protocol used. This path is the one assigned
+to the Ultralight 2.0 protocol
+
+* Also, the query parameters indicate the API Key defined in the first section, and the device ID defined in the previous
+one.
+
+* The payload of the request contains the measures, sepparated by bars. The names of the attributes used to report the
+measure are the ones defined in the `object_id` attributes of the device provisioning.
+
+* The `Content-Type` header is mandatory and **must** have the value `text/plain`.
+
+If everything is OK, the system will answer with a 200 OK code, and an empty answer.
```

Comment:
empty anser -> empty response?

Just a suggestion... actually I'm not sure what I prefer :)